Build failures in cborg with primitive-0.7.4.0

```
src/Codec/CBOR/Magic.hs:628:9: error:
    Ambiguous occurrence ‘copyPtrToMutableByteArray’
    It could refer to
       either ‘Prim.copyPtrToMutableByteArray’,
              imported from ‘Data.Primitive.ByteArray’ at src/Codec/CBOR/Magic.hs:112:1-77
           or ‘Codec.CBOR.Magic.copyPtrToMutableByteArray’,
              defined at src/Codec/CBOR/Magic.hs:660:1
    |
628 |         copyPtrToMutableByteArray (ptr `plusPtr` off) mba 0 len
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^
```

As far as I can tell v0.2.4.0 and up are affected and should be revised.

--- a/cborg/src/Codec/CBOR/Magic.hs
+++ b/cborg/src/Codec/CBOR/Magic.hs
@@ -109,7 +109,7 @@ import           Data.ByteString (ByteString)
 import qualified Data.ByteString          as BS
 import qualified Data.ByteString.Internal as BS
 import qualified Data.ByteString.Unsafe   as BS
-import           Data.Primitive.ByteArray as Prim hiding (copyByteArrayToPtr)
+import           Data.Primitive.ByteArray as Prim hiding (copyByteArrayToPtr, copyPtrToMutableByteArray)
 
 import           Foreign.ForeignPtr (withForeignPtr)
 import           Foreign.C (CUShort)

